src/api/source/mvt.js

```diff
@@ -0,0 +1,179 @@
+import * as rsys from '../../client/rsys';
+import Dataframe from '../../core/dataframe';
+import * as Protobuf from 'pbf';
+import { VectorTile } from '@mapbox/vector-tile';
+import { decodeLines, decodePolygons } from '../../client/mvt/feature-decoder';
+import TileClient from './TileClient';
+import Base from './base';
+import { RTT_WIDTH } from '../../core/renderer';
+
+// Constants for '@mapbox/vector-tile' geometry types, from https://github.com/mapbox/vector-tile-js/blob/v1.3.0/lib/vectortilefeature.js#L39
+const mvtDecoderGeomTypes = { point: 1, line: 2, polygon: 3 };
+
+const geometryTypes = {
+    UNKNOWN: 'unknown',
+    POINT: 'point',
+    LINE: 'line',
+    POLYGON: 'polygon'
+};
+
+export default class MVT extends Base {
+
+    /**
+     * Create a carto.source.MVT.
+     *
+     * @param {object} data - A MVT data object
+     * @param {object} metadata - A carto.source.mvt.Metadata object
```

Comment:
this object does not exist or is not available under the carto namespace.

I think we should update document and make public  [this object](https://github.com/CartoDB/carto-vl/blob/master/src/core/metadata.js)

Reply:
As commented in my previous comment, I left the Metadata object undocumented on purpose. For me, if this PR is merged, we will have an "experimental" MVT source. Not more, since we'll need to address some issues and I don't feel comfortable right now to write the API for the metadata object, I prefer to wait for that. That means that we shouldn't release a version with MVT support until these details are sorted out.

Reply:
ok, so I think is better to remove all the metadata references from the doc

Reply:
I undocumented the MVT class, since it is experimental